src/components/comment/Comment.styles.js

```diff
@@ -0,0 +1,50 @@
+export const styles = {
+  root: {
+    display: 'flex',
+    flexDirection: { xs: 'column', sm: 'row' },
+    alignItems: 'start',
+    gap: { xs: '12px', sm: '32px' },
+    borderBottom: '1px solid',
+    borderColor: 'primary.100',
+    py: '20px'
+  },
+  coopDetails: { color: 'primary.500', typography: { xs: 'subtitle2', sm: 'button' } },
+  description: { textAlign: 'start', display: 'flex', flexDirection: 'column', gap: '12px' },
+  comment: { typography: { xs: 'caption', sm: 'body1' } },
+  userInfo: {
+    root: {
+      display: 'flex',
+      flexDirection: { sm: 'column' },
+      minWidth: '130px',
+      gap: '8px'
+    },
+    img: {
+      width: { xs: '32px', sm: '48px' },
+      height: { xs: '32px', sm: '48px' },
+      border: '1px solid',
+      borderRadius: '50%',
+      borderColor: 'primary.900'
+    },
+    wrapper: {
+      display: 'flex',
+      flexDirection: 'column',
+      gap: '4px',
+      alignItems: 'start',
+      mb: 0,
```

Comment:
you use **"m"** one line below, i guess you don`t need **"mb"**

Reply:
no, ImageTitleWithDescriptions has m and mb in default styles, so to overnight them I should use both